### ISSUE TITLE
NumberInput is accepting alphabetic character

### ISSUE THREAD

**Issue by zavoyevatyel** [mantinedev/mantine#1379]
### What package has an issue

@mantine/core

### Describe the bug

Like a "123asd"

### In which browser did the problem occur

Google chrome

**Comment by rtivital** [comment]
Yes, it accepts any characters – it fixed everything on blur. It is required for custom formatter and decimal points to work. If this is unwanted behavior, use `<TextInput type="number" />` instead.
